lib/feedback.ml

```diff
@@ -42,7 +42,7 @@ type feedback_content =
   (* Extra metadata *)
   | Custom of Loc.t option * string * xml
   (* Generic messages *)
-  | Message of level * Loc.t option * Pp.t
+  | Message of level * Loc.t option * Quickfix.t list * Pp.t
```

Comment:
This new type seems strange.

Maybe introducing a new type of message `Quickfix` would work better; or even reusing `Custom` above.